src/css/profile/mobile/common/popup.less

```diff
@@ -192,7 +195,7 @@
 		height: auto;
 		margin-top: 8 * @px_base;
 		margin-left: 8 * @px_base;
-		color: var(--text-secondary-color);
+		color: var(--popup-secondary-color);
```

Comment:
I checked all guideline for find colors #999999 and #8f8f8f in context of dialog. Only place is calendar - labels for names of days. Could you direct which place in guideline refers to this change?
I think we don't need more variables for the same purpose.

Reply:
You can check it in GUI 1.7 version docs.
“4-5 Dialog_Body text 1 (Title + 2 level description)” on page 108 follows the rules in “14-2 Description_Text” on page 274.
In item 2, Light: #8f8f8f / Dark: #999999 is written.

Reply:
Ok. Thank you for explamation. I have only one comment. I think that name of variable should be: --popup-text-secondary-color